samples/kubeflow/README.md

```diff
@@ -53,6 +53,39 @@ cd samples/kubeflow/install/kubeflow
 
 cd -
 ```
+On an OpenShift 4+ cluster, follow [Kubeflow Documentation](https://www.kubeflow.org/docs/openshift/install-kubeflow/) to install Kubeflow with a few changes detailed below:
+```bash
+git clone https://github.com/opendatahub-io/manifests.git
+```
+
+Changes to `kfctl_openshift.yaml`:
+* Since we already have Istio installed, comment out `istio-crds` and `istio-install`
+* Comment out `cert-manager-crds` and `cert-manager` 
+* Comment out `seldon-core-operator` 
+
+A modified `kfctl_openshift.yaml` is provided in this repo.
+
+Make sure to update the manifest uri to your local file path
+```bash
+sed -i 's#uri: .*#uri: '$PWD'#' ./kfdef/kfctl_openshift.yaml
+```
+
+Changes to `kf-istio-resources.yaml`:
+```bash
+cd /opt/openshift-kfdef/kustomize/istio/base 
+```
+ *Replace sni_hosts with sniHosts on lines 63 and 96
+ 
+ Deploy Kubeflow resources:
+```bash
+cd /opt/openshift-kfdef
+sudo kfctl build --file=kfctl_openshift.yaml
+sudo kfctl apply --file=kfctl_openshift.yaml
+```
+Verify all Kubeflow resources are deployed:
+```bash
+kubectl get pods -n kubeflow
+```
```

Comment:
I think that it might be good to move this to a readme in the repo next to kfctl_openshift.yaml and just link from here as it adds too much noise